src/main/resources/caseIdForRetrigger.json

```diff
@@ -0,0 +1,7 @@
+{
+  "caseIdList": [
+    "1701946012369868",
+    "1701945970064792",
+    "1701945920181324"
+  ]
```

Comment:
I believe this file is also for testing in preview, so would you be updating this json file with prod case references?